README.md

```diff
@@ -1,19 +1,21 @@
-# 10up Scaffold
+10up Scaffold
+=====================
 
-## Local Dependencies
+At 10up, we strive to provide digital products that yield a top-notch user experience. In order to improve both our efficiency and consistency, we need to standardize what we use and how we use it. This theme scaffold allows us to share initial set up procedures to make sure all projects can get up and running as quickly as possible while closely adhering to 10up's high quality standards.
 
-`node`
+<a href="http://10up.com/contact/"><img src="https://10updotcom-uploads.s3.amazonaws.com/uploads/2016/08/10up_github_banner-2.png"></a>
 
-`npm`
+## Dependencies
 
-`gulp`
-
-`webpack`
+1. [.editorconfig](http://editorconfig.org/) Plugin - The .editorconfig file ensures that everyone contributing to the project will use the same indentation.
+2. [Node & NPM](https://www.npmjs.com/get-npm) - Build packages and 3rd party dependencies are managed through NPM, so you will need that installed globally
+3. [Gulp](https://gulpjs.com/) - Gulp is used as the main task runner, it runs PostCSS, image processed, handles SVG sprites (if needed), and executes Webpack
```

Comment:
@timwright12  "image processed" should be "processes images" to match the tense of the rest of the sentence.